store/tikv/2pc.go

```diff
@@ -423,26 +432,30 @@ func (c *twoPhaseCommitter) commitSingleBatch(bo *Backoffer, batch batchKeys) er
 }
 
 func (c *twoPhaseCommitter) cleanupSingleBatch(bo *Backoffer, batch batchKeys) error {
-	req := &pb.Request{
-		Type: pb.MessageType_CmdBatchRollback,
-		CmdBatchRollbackReq: &pb.CmdBatchRollbackRequest{
+	req := &tikvrpc.Request{
+		Type: tikvrpc.CmdBatchRollback,
+		BatchRollback: &pb.BatchRollbackRequest{
 			Keys:         batch.keys,
 			StartVersion: c.startTS,
 		},
 	}
-	resp, err := c.store.SendKVReq(bo, req, batch.region, readTimeoutShort)
+	resp, err := c.store.SendReq(bo, req, batch.region, readTimeoutShort)
```

Comment:
Usually read timeout is more important than write timeout, since the write impl simply puts data into kernel buffer. And there are occasions that different timeouts are needed for reading, so there are short/long read timeouts.